### PR TITLE
Fix 'or' in controls 5.1 and 5.2

### PR DESCRIPTION
**Pull request by emilyh315**
'only_if' tests in controls 5.1 and 5.2 were not executing as expected due to incorrect 'or' logic. The test was only checking for equality with the first item after '==.' Both tests were getting skipped on my machine despite the fact that my os family came up as 'redhat.' I picked a way to rewrite the test to incorporate the correct logic, but there are other ways it could be written.

### CODE REVIEW

File: controls/docker_level2.rb [hunk 1]
```diff
@@ -163,7 +163,7 @@
   ref 'https://docs.docker.com/engine/reference/run/#security-configuration'
   ref 'http://wiki.apparmor.net/index.php/Main_Page'
 
-  only_if { ['ubuntu', 'debian'].include? os[:family] }
+  only_if { %w(ubuntu debian).include? os[:family] }
```
Comment (chris-rock) [inline controls/docker_level2.rb]: `os[:family]` will be identical for ubuntu and debian. We should use `os.name` instead

**Comment by chris-rock** [comment]
@emilyh315 Thank you for highlighting this issue. We are currently working on ideas to automatically test the implementation against real systems. This would help us to identify those issues. 

InSpec prefers `os.name` over `os.family` in cases where we want to be specific. I think it makes sense to use it here.

Please do me a favor to squash & sign-off your commits.

**Comment by chris-rock** [comment]
@emilyh315 Something went wrong with the rebase. Seems like the master got merged into the branch. Can you use `git rebase -i master`?

**Comment by emilyh315** [comment]
Something did go wrong when I was trying to squash. I think the branch is all wrong now. It might be easiest to just close this and I can open a new pull request with a single commit and sign it.

**Pull request closed without merging**